packages/components/scripts/storybook/node-modify-page-title.cjs

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Change the generated index.html document title to "Solid Design System by Union Investment | Version: x.x.x".
```

Comment:
I get the intention of the change and like the output – but this might lead to problems:

Our x.x.x URL resolves to different versions, but OG always picks up the version of the moment being sent, so when we link something on the latest branch, it might always stay at "1.4.2" – and people will say "ah, I thought this is only this version".

That might lead to confusion – and there already is often enough confusion around those Storybook URLs, I'm sure, that it will just increase...

For fixed Version URLs, this is great, for sure.

----

Therefore: I would suggest to NOT rewrite this for Open Graph on Build Time but only on Runtime via Vite + JS and even saying "Latest Storybook" in the Browser instead of the version on this particular URL. So people see it in the browser always correct (1.2.4 Docs, 1.2.x docs, latest docs etc.)

Reply:
I see the point with the wildcard folders. TBH I did not think about them at this point. I will have a look on it and see if I find a solution to rewrite the title according to the version the link contains (fixed versions, latest and wildcard folders).

In general, this title is only used for sharing purpose as Storybook rewrites the title on runtime anyways.

Reply:
If this is only for sharing purposes we would have to rewrite it before/for every CDN folder individually.

I definitely wouldn't recommend the current solution but would rather create an issue for that, so we can have a concept, that helps without possible confusion.

Reply:
not for every folder but for the wildcard folder and as stated before, I did not have them in mind and was not able to find a way to get this done.

it would work for latest, fixed version and feature branches but not for the wildcard folders. Unfortunately this is probably the more important ones next to latest...